### ISSUE TITLE
[FR]: Visual indicator or toogle button to inform that the left pane is being filtered

### ISSUE THREAD

**Issue by starbuck3000** [martinrotter/rssguard#873]
### Brief description of the feature request

This feature would allow users to be informed that the current view shown in the left pane (categories and feeds) is being filtered in some way and that some items may be hidden. 

An initial version could consist in a text mention next to the account label, like as follows:
"my feeds (RSS/ATOM/JSON)" would become "my feeds (RSS/ATOM/JSON) \**filtered*\*"

A second version could actually indicate the type of filtering that is enforced:
"my feeds (RSS/ATOM/JSON)" would become "my feeds (RSS/ATOM/JSON) \**unread only*\*"

Finally, the best option (in my opinion) would be to have an icon at the top of the feeds pane that allows the user to toogle the native filters available (e.g., all, unread only, important only).

Thanks

**Comment by martinrotter** [comment]
Yes, some kind of clear visual indication would make sense. Will do that sometimes in future.

**Comment by starbuck3000** [comment]
thank you!
